images/cass-config-builder/tests/smoke-test.sh

```diff
@@ -36,15 +34,14 @@ kubectl wait --for=condition=ready pod --selector app.kubernetes.io/name=k8ssand
 #reapply with new config
 current_config_builder=$(kubectl get configmap k8ssandra-operator-cass-operator-manager-config -n k8ssandra-operator -o jsonpath='{.data.image_config\.yaml}' | grep 'config-builder' | awk '{print $2}')
 echo $current_config_builder
-kubectl get configmap k8ssandra-operator-cass-operator-manager-config -n k8ssandra-operator -o yaml | \
-  sed "s|config-builder: ${current_config_builder}|config-builder: ${IMAGE_NAME}|" | \
-  kubectl apply -f -
+kubectl get configmap k8ssandra-operator-cass-operator-manager-config -n k8ssandra-operator -o yaml |
+	sed "s|config-builder: ${current_config_builder}|config-builder: ${IMAGE_NAME}|" |
```

Comment:
I think you'll still need the `\` at the end of the line here, no?

Reply:
this was migrated verbatim from the existing tests

Reply:
it seems like they're effectively the same, the `\` after a `|` is purely for style reasons. TIL!

Reply:
TIL too